teuthology/suite/run.py

```diff
@@ -305,7 +305,7 @@ def choose_teuthology_branch(self):
             if not teuthology_branch:
                 teuthology_branch = actual_branch
             teuthology_sha1 = util.git_ls_remote(
-                f"file://{config.teuthology_path}",
+                f"file://$(realpath {config.teuthology_path})",
```

Comment:
I would much rather see something like [pathlib](https://docs.python.org/3/library/pathlib.html#expanding-and-resolving-paths) used here :)